manifests/modelserving/ovms-gpu-ootb.yaml

```
@@ -8,6 +8,7 @@ metadata:
   annotations:
     tags: 'ovms,servingruntime'
     description: 'OpenVino with GPU Support Model Serving Definition'
+    opendatahub.io/modelServingSupport: "['single', 'multi']"
```

I would say always use the single quote at the outer layer. In this case, it will be translated to `'[''single'', ''multi'']'` but not `'["single", "multi"]'`
```suggestion
    opendatahub.io/modelServingSupport: '["single", "multi"]'
```

---

Yes, you are completely right, damn, I missed this, thanks a lot!